airbyte-webapp/src/types/global-fetch.d.ts

```diff
@@ -0,0 +1 @@
+declare type GlobalFetch = WindowOrWorkerGlobalScope;
```

Comment:
What does this do?

Reply:
It may not be needed any longer, but this wasn't typed globally the way it should be.